### PR TITLE
change for multinode training

### PR DESCRIPTION
**Pull request by kurarrr**
https://github.com/blue-oil/blueoil/issues/727

This PR includes two changes.

- Support mpirun or horovodrun command on k8s job
- Support _TFDSReader in multinode

## How has this been tested?

Tested in 2 pods in GKE.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: lmnet/lmnet/utils/horovod.py [hunk 1]
```diff
@@ -61,6 +61,8 @@ def _get_pname(pid):
 
 # return True if the parent process is mpirun or horovodrun
 def is_enabled():
+    if os.getenv("USE_HOROVOD"):
```
Comment (hadusam) [inline lmnet/lmnet/utils/horovod.py]: IMHO: How about `ENABLE_HOROVOD`?

Comment (kurarrr) [reply lmnet/lmnet/utils/horovod.py]: I named this environment variable after `use_horovod` variable in python, but `ENABLE_HOROVOD` also seems fine.

Comment (hadusam) [reply lmnet/lmnet/utils/horovod.py]: Oh, OK, there is a variable named `use_horovod` so it is fine `USE_HOROVOD`. Thanks.

**Comment by hadusam** [comment]
@tkng @tsawada 
Could you review this PR for python readability? Thanks!

**Pull request merged**